docs/usage/trusted-tls-for-control-planes.md.md

```diff
@@ -0,0 +1,48 @@
+# Trusted TLS certificate for shoot control planes
```

Comment:
Rename the file from `trusted-tls-for-control-planes.md.md` to `trusted-tls-for-control-planes.md` :)